test/helpers/vagrant.go

```diff
@@ -21,12 +21,15 @@ import (
 	"os/exec"
 	"strings"
 
+	"github.com/cilium/cilium/test/config"
+
 	"github.com/onsi/ginkgo"
 	"github.com/sirupsen/logrus"
 )
 
-//Create a new vagrant server. Receives and scope that it's the target server that need to be created.
-// In case of any error on vagrant [provision|up|ssh-config] error will be returned.
+// CreateVM a new vagrant server. Receives and scope that it's the target
```

Comment:
creates a new...

Receives a scope which indicates the target server that needs to be created.